Informatics II/003/Task 1/Perytron Solution/task1.c

```diff
@@ -0,0 +1,67 @@
+#include <stdio.h>
+#include <time.h> // measuring run time
+
+int n, t;
+int A[100000000];
+
+int linear_search(int A[], int n, int t) {
+    for (int i = 0; i < n; i++) {
+        if (A[i] == t) {
+            // return i;
+            return 1;
+        }
+    }
+    // return -1;
+    return 0;
+}
+
+int binary_search(int A[], int l, int r, int t) {
+    /* Binary search adjust its boundaries until there is only one element left
+    (Divide & Conquer). If the searched element is not in the array, it will
+    cross-over the boundaries (because we add/subtract 1 to the middle) and
+    the following condition turns true.*/
+    if (l > r) {
+        // return -1;
+        return 0;
+    }
+    else {
```

Comment:
crazy weird bracket formatting 😅

Reply:
Oh! I've disabled the default formatter in vscode because it used to put the brackets to the next row instead of the end of the current line. This is a side effect, because it does not automatically change such weird stuff. 😆

Reply:
Huh! The default formatter does it like this! When saving the file, it automatically jumps to the next line!